.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: go
+go:
+  - "1.12"
+
+git:
+  depth: 1
+
+services:
+  - docker
+
+script:
+  - make test
+  - curl -sL https://git.io/goreleaser | bash -s -- --snapshot --skip-publish --rm-dist
+  - docker image ls
```

Comment:
Is it necessary?

Reply:
Nope. It was there just for debugging. I removed it.